torchprime/launcher/thunk.py

```diff
@@ -14,6 +14,12 @@
 # `--megascale_grpc_enable_xor_tracer=false` flag when libtpu is updated
 xla_flags = os.environ.get("LIBTPU_INIT_ARGS", "")
 xla_flags = f"{xla_flags} --megascale_grpc_enable_xor_tracer=false"
+
+# Workaround for MegaScale perf regression
+#
+# TODO(b/423386767): Remove the `--megascale_grpc_num_channels` override
```

Comment:
nit: could you please file a github issue with some background for this and cross reference internal bug